### PR TITLE
Adding Hinton knowledge distillation

### PR DESCRIPTION
**Pull request by RajGhugare19**

### CODE REVIEW

File: trainer.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
+from comet_ml import Experiment
```
Comment (akshaykvnit) [inline trainer.py:1]: this file is extra. Please remove it.

**Review by akshaykulkarni07 — CHANGES_REQUESTED** [review]
Remove the extra file. Rest seems fine, but I'll still request @navidpanchi to just go through once.

**Comment by akshaykvnit** [comment]
Another thing @RajGhugare19, you have to update the README file (see how other methods are mentioned in the main README and do the same).

**Comment by RajGhugare19** [comment]
I have deleted the extra file and updated the README @akshaykvnit

**Review by navidpanchi — APPROVED** [review]

**Pull request merged**